src/js/fetchTimeline.js

```diff
@@ -0,0 +1,16 @@
+const fetchTimeline = () => {
+    
+    return fetch("http://localhost:8080/api/1.0/twitter/timeline")
```

Comment:
since fetch itself is a promise if you were to chain then within the fetch, you could run into odd race conditions.  Would be cleaner to return your own promise explicitly as then you can ensure when data is returned.